pkg/config/config.go

```diff
@@ -212,3 +217,8 @@ type Exec struct {
 
 	Tags map[string]string `name:"tag" def:"" desc:"tag in key=value form. The flag may be specified multiple times" mapstructure:"tags"`
 }
+
+type Admin struct {
+	// TODO: use <storagePath> somehow
+	SocketPath string `def:"/tmp/pyroscope.sock" desc:"path where the admin UDS will be created." mapstructure:"admin-socket-path"`
```

Comment:
If I understand this right, this will look like this in the config file:
```
admin:
  admin-socket-path: 
```

I think we can drop `admin-` prefix, because it will be clear from context in this case

Reply:
Also, I don't know what UDS stands for. We should probably spell it out more, because it's not super clear

Reply:
Good points, renamed the property and removed UDS from there since it's an implementation detail. Also explained what UDS stands for in the pkg code.

It means Unix Domain Socket BTW